src/__tests__/components/NavBar/EditorDesignName.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import EditorDesignName from "../../../components/NavBar/EditorDesignName";
```

Comment:
You can use @ to import EditorDesignName

Reply:
update it.